// /***
//  * Estrutura básica de uma classe de exceção customizada!
//  */
// class MinhaExcecao extends Error {
//   constructor(message: string) {
//     super(message);
//     this.name = 'MinhaExcecao';
//   }
// }

// function exemplo() {
//   throw new MinhaExcecao('Está funcionando meu erro customizado!')
// }

// exemplo()

// console.log('Esta linha nunca será executada! Exceções terminam a execução do programa se não tratadas!')